tests/test_analysis/test_count_in_voxel.py

```diff
@@ -0,0 +1,62 @@
+import unittest
+from pytraj import *
+import pytraj as pt
+from utils import fn
+from pytraj.testing import aa_eq
+import numpy as np
+
+
+# tests to see if function can find an atom if we search with a coordinate
+# very close to the atom's xyz coordinates
+def search_for_atom(self):
```

Comment:
oh, for each test, you need to add `test_` prefix (`test_search_for_atom`), otherwise, `pytest` won't be recognise it.

Reply:
https://github.com/Amber-MD/pytraj/actions/runs/6424481585/job/17445206646

e.g: click "Test with pytest" and search for your function's names. They are not in there.

Reply:
Cool, got the tests to work by adding the prefix and using your `python3 -m pytest` command